nltk/classify/decisiontree.py

```diff
@@ -77,6 +77,7 @@ def pretty_format(self, width=70, prefix='', depth=4):
         decision tree.  Each line in this string corresponds to a
         single decision tree node or leaf, and indentation is used to
         display the structure of the decision tree.
+        rtype : str
```

Comment:
Please change to `:rtype: str`

Reply:
Thank you for your early reply. How do I modify the current commit without sending another pull request?